lib/jsdom/living/helpers/iterable-weak-set.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+// This is a static function to improve memory usage when multiple
+// `IterableWeakSet` instances are created, so that each `FinalizationRegistry`
+// uses the same shared function:
```

Comment:
If this is the real reason then I'll just convert it to a closure as this is a fully general argument against any use of functions inside functions and it's not worth departing from the usual readable style.